heroku/cli
# refactor(spaces): Move command spaces:wait to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001b1QMeYAM/view

This converts `spaces:wait` to oclif and its associated tests.

### Testing
1) Pull down this branch and run `yarn build`
2) Create a space
3) While this space is allocating, run `./bin/run spaces:wait NEW_SPACE_NAME`
5) Wait until it allocates (might take up to 10 mins)
6) Verify the command succeeds
6) Compare with prod
